packages/blade/src/components/TopNav/TopNav.web.tsx

```diff
@@ -0,0 +1,140 @@
+        stroke="url(#paint0_linear_89426_105396)"
+        strokeMiterlimit="10"
+      />
+      <defs>
+        <linearGradient
+          id="paint0_linear_89426_105396"
+          x1="162.854"
+          y1="63.9672"
+          x2="160.504"
+          y2="-544.656"
+          gradientUnits="userSpaceOnUse"
+        >
+          <stop stopColor="#6F9ADA" />
+          <stop offset="1" stopColor="white" />
+        </linearGradient>
+      </defs>
+    </svg>
+  );
+};
+
+type TopNavProps = {
+  children: React.ReactNode;
+} & Pick<
+  BoxProps,
+  | 'padding'
+  | 'paddingTop'
+  | 'paddingBottom'
+  | 'paddingLeft'
+  | 'paddingRight'
+  | 'paddingX'
+  | 'paddingY'
+  | 'backgroundColor'
+  | 'position'
+  | 'top'
+  | 'bottom'
+  | 'left'
+  | 'right'
+  | 'width'
+  | 'zIndex'
+>;
+
+const TopNav = ({ children, ...styledProps }: TopNavProps): React.ReactElement => {
+  return (
+    <Box
+      display="grid"
+      gridTemplateColumns={{ base: 'minmax(0, 1fr) auto', m: 'auto minmax(0, 1fr) auto' }}
+      alignItems="center"
+      position="sticky"
+      top="0px"
+      width="100%"
+      paddingX="spacing.3"
+      height={makeSize(TOP_NAV_HEIGHT)}
+      backgroundColor="surface.background.gray.subtle"
+      zIndex={1}
+      {...styledProps}
+    >
+      {children}
+      <Box position="absolute" top="0px" left="0px" pointerEvents="none">
+        <RazorpayLinesSvg />
+      </Box>
+    </Box>
+  );
+};
+
+const TopNavBrand = ({ children }: { children: React.ReactNode }): React.ReactElement => {
+  return (
+    <Box
+      display={{
+        base: 'none',
+        m: 'flex',
+      }}
+      flexDirection="row"
+      marginTop="spacing.4"
+      width={{
+        base: makeSize(EXPANDED_L1_WIDTH_MOBILE),
+        xl: makeSize(EXPANDED_L1_WIDTH_DESKTOP),
+      }}
+    >
+      <BaseBox width="100%" textAlign="center">
+        {children}
+      </BaseBox>
+      <Box alignSelf="center" display={{ base: 'none', m: 'block' }}>
+        <Divider
```

Comment:
The spacing on left of Home icon and right of icon seem uneven 🤔 
<img width="231" alt="image" src="https://github.com/razorpay/blade/assets/30949385/60111430-5c7f-4197-9490-e2c0a6b7fddf">

Reply:
This is expected, it has a left margin, otheriwise when selected the white background will be too close to the divider.